config/config.exs

```diff
@@ -1,12 +1,20 @@
 import Config
 
-config :beacon, ecto_repos: [Beacon.Repo]
+config :beacon, ecto_repos: [Beacon.BeaconTest.Repo]
```

Comment:
I think in Oban, there is no `dev.exs` or `test.exs` because all the config is only for local testing.  This was probably why I put it in `config.exs` in hopes that we could also move towards that pattern?

Reply:
Makes sense. Better have one file than two or more.